typescript-intro/typescript-intro-quiz-notes.md

```diff
@@ -0,0 +1,51 @@
+# typescript-intro-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- When was TypeScript introduced?
+  October 2012
+
+- Who created TypeScript?
+  Microsoft
+
+- What does the TypeScript `Compiler` do?
+
+addresses runtime errors resulting from JavaScript’s lack of type checking, it gives JS programmers access to tools providing benefits of typed language to turn runtime errors (found while program is running) into compile-time errors (found before running a program)
```

Comment:
I would say that's what TypeScript itself does, but the compiler specifically is just responsible for converting TypeScript code to JavaScript code.